src/Formatter/HumanReadableExceptionFormatter.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace PolderKnowledge\LogModule\Formatter;
+
+use Monolog\Formatter\FormatterInterface;
+use Monolog\Formatter\NormalizerFormatter;
+
+/**
+ * Format an Exception in a similar way PHP does by default when an exception bubbles to the top
+ */
+class HumanReadableExceptionFormatter extends NormalizerFormatter implements FormatterInterface
+{
+    public function format(array $record): string
+    {
+        //var_dump($record);exit;
+        $exception = $record['context']['exception'] ?? null;
+        if ($exception) {
+            return $this->printFromException($exception);
+        } else {
+            return $this->printWithoutException($record);
+        }
+    }
+
+    protected function printWithoutException(array $record): string
+    {
+        return sprintf('[%s] %s: %s',
+            date('r'),
+            $record['level_name'],
+            $record['message']
+        );
+    }
+    
+    protected function printFromException(\Throwable $exception)
+    {
+        return implode("\n", $this->linesFromException($exception)) . "\n"
+            . "---------------------------------------\n";
+    }
+
+    protected function linesFromException(\Throwable $exception, bool $nested = false): array
+    {
+        $lines = [];
+        $lines[] = $this->printFirstLineFromException($exception, $nested);
+
+        foreach ($exception->getTrace() as $traceLine) {
+            $lines[] = $this->formatTraceLine($traceLine);
+        }
+
+        if ($exception->getPrevious()) {
+            $previousLines = $this->linesFromException($exception->getPrevious(), true);
+
+            // recursion makes sure deeper nesting has longer prefixes
+            foreach ($previousLines as $previousLine) {
+                $lines[] = '……' . $previousLine;
```

Comment:
think I would prefer whitespace indenting here. Makes it more readable?

Reply:
Ok I don't really care as long as it's indented